assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -347,6 +347,11 @@ che.infra.kubernetes.workspace_start_timeout_min=8
 # Defines the timeout in minutes that limits the period for which Kubernetes Ingress become ready
 che.infra.kubernetes.ingress_start_timeout_min=5
 
+# If during workspace startup an unrecoverable event defined in 'che.infra.kubernetes.workspace_unrecoverable_events' occurs
+# terminate workspace immediately instead of waiting util timeout
+che.infra.kubernetes.workspace_fail_on_unrecoverable_events=true
```

Comment:
Can you describe why it is necessary to have separate configuration property for disabling listening to unrecoverable events. Maybe it makes sense to configure empty `che.infra.kubernetes.workspace_unrecoverable_events` for disabling instead of having a separate one?

Reply:
good point, less config we have the better